### DISCUSSION TITLE
Guys i f*** up...I must go back to NC stable version from beta asap

### DISCUSSION THREAD

**Discussion by megatonix** [nextcloud/all-in-one#2862] [Questions] [answered]
I have problem with login to my nextcloud about 2 weeks ago. I found that maybe i must change to version beta and update, so i did it. But i do it on my production server and now i have problems with opening Files, login and other things. So my question is... How i can go back to stable version  eg. 26.0.1 grom my Version 27.0.0 RC4

**Comment by szaimen** [comment, marked as answer]
Hi, you can restore a backup from before the updated. Obviously you will lose any changed that were made in the meantime.

**Reply by megatonix** [reply]
I done that but version is still 27 0.0 RC4

**Reply by szaimen** [reply]
> I done that but version is still 27 0.0 RC4

Are you completely sure that you actually restored a backup from before upgrading?

**Reply by megatonix** [reply]
Not on 100%, i can try one more time and i will post comment here

**Reply by megatonix** [reply]
But Backup not only restore my data? How Backup restore version on whole NextCloud?

**Reply by szaimen** [reply]
No, it should restore everything including whole Nextcloud.

**Reply by megatonix** [reply]
Okey, i will try it one more time

**Reply by megatonix** [reply]
I have one more question. If i have version 27 0.0. RC4 can i wait untill new release of Nextcloud and then my NC will update to stable ? Or this not working like that ?

**Reply by szaimen** [reply]
> If i have version 27 0.0. RC4 can i wait untill new release of Nextcloud and then my NC will update to stable ? Or this not working like that ?

You will need to upgrade manually via `sudo docker exec --user www-data -it nextcloud-aio-nextcloud bash -c "php updater/updater.phar --no-interaction && php occ app:enable nextcloud-aio --force"` until we release a new version that includes Nextcloud 27 for everyone. It will likely be 27.0.1 or 27.0.2. See https://github.com/nextcloud/all-in-one/issues/2686

**Reply by megatonix** [reply]
Ok, thanks for answer
